metis-enrichment/metis-enrichment-client/pom.xml

```diff
@@ -54,16 +54,7 @@
             <artifactId>metis-dereference-common</artifactId>
             <version>${project.version}</version>
         </dependency>
-		<dependency>
-		    <groupId>org.slf4j</groupId>
-		    <artifactId>slf4j-simple</artifactId>
-		    <version>${version.slf4j}</version>
-		</dependency>
-		<dependency>
-		    <groupId>org.slf4j</groupId>
-		    <artifactId>slf4j-jdk14</artifactId>
-		    <version>${version.slf4j}</version>
-		</dependency>
```

Comment:
We shouldn't be moving to log4j, so these dependencies should stay.

Reply:
Is there a specific reason why we've decided to stick with SLF4J while simultaneously using Log4J2 as the logging implementation? Log4J2 offers so much more functionality that we can't use if we stick with SLF4J.

Is this particularly because we are using dependencies throughout Metis that rely on the interface provided by SLF4J?

Reply:
Well, if we do decide to switch from one to the other, it should be a decision that we talked about, agreed on and subsequently implemented for all tools (not just enrichment). There are most likely also arguments to be made for sticking with SLF4J.

Regardless, though, I would say that this is outside the scope of this ticket, therefore I think this should be reverted to the way it was (using SLF4J).

Reply:
Done.